src/pcapi/utils/object_storage.py

```diff
@@ -78,3 +90,11 @@ def build_thumb_path(pc_object: Model, index: int) -> str:
         + humanize(pc_object.id)
         + (("_" + str(index)) if index > 0 else "")
     )
+
+
+def _get_gcp_storage_client_bucket() -> Bucket:
+    gcp_credentials_as_string = settings.GCP_BUCKET_CREDENTIALS
+    credentials = Credentials.from_service_account_info(gcp_credentials_as_string)
+    storage_client = Client(credentials=credentials, project=gcp_credentials_as_string.get("project_id"))
```

Comment:
```suggestion
    credentials = Credentials.from_service_account_info(settings.GCP_BUCKET_CREDENTIALS)
    project_id = settings.GCP_BUCKET_CREDENTIALS.get("project_id")
    storage_client = Client(credentials=credentials, project=project_id")
```

puisque ça n'est plus une string mais un dict, désormais.